frontend-react/e2e/mocks/appInsightsConfig.json

```diff
@@ -0,0 +1,43 @@
+{
+    "version": "1.0.0",
+    "enabled": true,
+    "featureOptIn": {
+        "iKeyUsage": {
+            "mode": 3,
+            "onCfg": {
```

Comment:
might good to add some comments here, either at the top of the file or inline, about what these obtuse json key/value pairs are doing, like `"throttleMgrCfg.109.disabled": false,`

Reply:
i honestly don't know what it's doing. it's simply a copy+paste from a real response.

Reply:
Also, comments aren't allowed in JSON